<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MktShowParamsDlg</class>
 <widget class="QDialog" name="MktShowParamsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="control::MwcPushButtonNormal" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>325</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Skip the backup. I understand that I can loose my funds.</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="applyButton">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>325</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Backup the data</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::MwcComboBox" name="feeLevel">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>110</y>
     <width>311</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Time reserved to redeem the coins</string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>110</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Fee Level:</string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>170</y>
     <width>126</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Show offers:</string>
   </property>
  </widget>
  <widget class="control::MwcComboBox" name="buySellCombo">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>170</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Currency that you want to swap with</string>
   </property>
   <item>
    <property name="text">
     <string>Buy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sell</string>
    </property>
   </item>
  </widget>
  <widget class="control::MwcComboBox" name="secCurrencyCombo">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>170</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Currency that you want to swap with</string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_4">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>170</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>MWC  for</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_5">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>245</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Range (optional):</string>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="rangeMin">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>240</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Number of confirmations for MWC coin</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>512</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>245</y>
     <width>46</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="rangeMax">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Number of confirmations for MWC coin</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>512</number>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>51</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>MWC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>461</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Offers to view</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcComboBox</class>
   <extends>QComboBox</extends>
   <header>control_desktop/MwcComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
